students/M1lya1/hw-1/script.js

```diff
@@ -0,0 +1,6 @@
+document.querySelector('button').addEventListener('click', () => {
+
+    const div = document.querySelector('div');
+
+    div.innerHTML = `hello student`;
+})
```

Comment:
Please add a new line at the bottom of your files.
Why? [link](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)
How to? [link](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files)